README.md

```diff
@@ -35,195 +26,99 @@ https://simplereport.gov/
 ```
-REACT_APP_BACKEND_URL=http://localhost:8080
-REACT_APP_BASE_URL=http://localhost:3000
-REACT_APP_OKTA_ENABLED=true
+"The build failed because the process exited too early. This probably means the system ran out of memory or someone called `kill -9` on the process."
 ```
 
-### Updating user role
-
-By default the local test user is an organization admin role. If you need to change this value to test out other permissions.
-It can be set in `application-local.yaml`. If you have not created one run:
-
-bash
-
-```
-touch backend/src/main/resources/application-local.yaml
-```
+How to update resources limits on your [Mac](https://docs.docker.com/desktop/mac/#resources), and [Windows](https://docs.docker.com/desktop/windows/#resources) machines.
 
-#### Organization roles
+## Running outside of docker
 
-Organization roles can be set by adding the following to `application-local.yaml`:
+If you wish to run outside of docker-compose (bare metal), you'll need to install all the dependencies locally.
+See more [info here](https://github.com/CDCgov/prime-simplereport/wiki/Running-outside-of-docker).
 
-```
-simple-report:
-  demo-users:
-    default-user:
-      authorization:
-        granted-roles: ADMIN
-```
+## Support admin
 
-current role types are `ADMIN`, `USER`, `ENTRY_ONLY`, `ALL_FACILITIES`, and `NO_ACCESS`. You can check `backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java` for a list of available roles
+You can make your user a support admin by assigning yourself the `SR-DEV-ADMINS` group in Okta Preview.
 
-`ADMIN` - an organization admin with full access to their organization
-`USER` - a site user the has access to everything in their organization but the gear icon
-`ENTRY_ONLY` - a site user that only has access to the Conduct Test tab
-`ALL_FACILITIES` - a site user that can access all facilities in their organization
-`NO_ACCESS` - a member of an organization who has no permissions without possessing other roles
+Support admins can access the `/admin` paths and support admin APIs.
 
-#### Site roles
+## Roles
 
-You can make the default user a site admin by adding the following to `application-local.yaml`:
+The available user role types are `ADMIN`, `USER`, `ENTRY_ONLY`, `ALL_FACILITIES`, and `NO_ACCESS`. You can check `backend/src/main/java/gov/cdc/usds/simplereport/config/authorization/OrganizationRole.java` for a list of available roles
 
-```
-simple-report:
-  demo-users:
-    site-admin-emails:
-      - bob@example.com
-```
+- `ADMIN` - an organization admin with full access to their organization
+- `USER` - a site user the has access to everything in their organization but the gear icon
+- `ENTRY_ONLY` - a site user that only has access to the Conduct Test tab
+- `ALL_FACILITIES` - a site user that can access all facilities in their organization
+- `NO_ACCESS` - a member of an organization who has no permissions without possessing other roles
```

Comment:
Could you add that all members of an org are part of the no-access group? That's how we get user lists for the organization.